src/rdkafka.h

```diff
@@ -3762,7 +3762,8 @@ rd_kafka_consumer_group_metadata (rd_kafka_t *rk);
  *         rd_kafka_consumer_group_metadata_destroy().
  */
 RD_EXPORT rd_kafka_consumer_group_metadata_t *
-rd_kafka_consumer_group_metadata_new (const char *group_id);
+rd_kafka_consumer_group_metadata_new (const char *group_id,
```

Comment:
oh, right. is this public because of testing? i was thinking it was private because it's not necessary to expose. we're going to have .._new3 and ..new4 before we're done with this...

Reply:
oh, you say it's typically used for writing tests in the comment. don't mind me.